tests/unit/CardUtilsTest.ts

```diff
@@ -200,6 +204,28 @@ const cardFeedsCollection: OnyxCollection<OnyxTypes.CardFeeds> = {
     },
 };
 
+/* eslint-disable @typescript-eslint/naming-convention */
+const allCardsList = {
+    'cards_11111111_oauth.capitalone.com': directFeedCardsMultipleList,
+    cards_11111111_vcf: customFeedCardsList,
```

Comment:
Is this on purpose not a string? 

Also lets test `vcf1` since that is what we use now
```suggestion
    'cards_11111111_vcf1': customFeedCardsList,
```

Reply:
> Is this on purpose not a string?

Yeah, there is a prettier error
![image](https://github.com/user-attachments/assets/25b9e8db-214c-491d-b5a7-ad148971eb6b)

> Also lets test vcf1 since that is what we use now

Updated!